qabelbox/src/main/java/de/qabel/qabelbox/identity/view/adapter/IdentityDetailsAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package de.qabel.qabelbox.identity.view.adapter
+
+import android.view.View
+import de.qabel.core.config.Identity
+import de.qabel.core.ui.initials
+import de.qabel.core.ui.readableKey
+import de.qabel.core.ui.readableUrl
+import de.qabel.qabelbox.contacts.extensions.color
+import de.qabel.qabelbox.contacts.view.widgets.IdentityIconDrawable
+import kotlinx.android.synthetic.main.fragment_identity_details.view.*
+
+
+class IdentityDetailsAdapter() {
+
+    var view: View? = null
+
+    fun loadIdentity(identity: Identity) {
+        view?.apply {
+            val mail = identity.email ?: ""
+            val phone = identity.phone ?: ""
+
+            identity_icon.background = IdentityIconDrawable(
+                    text = identity.initials(),
+                    color = identity.color(context))
+            identity_initial.text = identity.initials()
+            editIdentityAlias.text = identity.alias
```

Comment:
I like that its camelCase but for the sake of consistency, you should probably stick to snake_case for the view files and use `edit_identity_alias`. If it's practical to use camelCase there then it should be decided as a group and all occurrencies should be converted to camelCase (like it happening with the java -> kotlin conversion) but this mixed style is not good.